python/tests/catalog/test_glue.py

```diff
@@ -0,0 +1,77 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+import getpass as gt
+import random
+import string
+
+import pytest
+
+from pyiceberg.catalog.glue import GlueCatalog
+from pyiceberg.exceptions import NoSuchNamespaceError
+from pyiceberg.schema import Schema
```

Comment:
For testing we could also use moto: https://docs.getmoto.org/en/latest/docs/services/glue.html

What do you think @jackye1995 ?

Reply:
for the scope of this PR it is enough, although `update_table`, one of the most needed API is not supported there yet... It seems straightforward to contribute though. We can probably consider adding that support directly in the repo.